### PR TITLE
Lag suffix, can be applied to an emote to save space

### PR DESCRIPTION
**Pull request by WhiteNervosa**
The lag suffix can be changed to just be an emote animation on one frame to save space

### CODE REVIEW

File: assets/chat/css/generify.scss [hunk 1]
```diff
@@ -11,6 +11,22 @@
     margin-bottom: -7px;
 }
 
+// LAG any emote or move to anim and save space
+// by using it on one frame
+
+@keyframes lag-anim {
```
Comment (jbpratt78) [inline assets/chat/css/generify.scss]: LGTM

File: assets/chat/js/formatters.js [hunk 2]
```diff
@@ -374,6 +374,7 @@ class UrlFormatter {
                 url = self.encodeUrl(m[0]);
                 const extra = self.encodeUrl(decodedUrl.substring(m[0].length));
                 const href = scheme + url;
+                url = href.replace(/(^\w+:|^|www.)|\/\//g, ''); // Removes https://www. from links in chat
```
Comment (jbpratt78) [inline assets/chat/js/formatters.js]: I am not interested in this feature. Others may have interest in it though.

Comment (WhiteNervosa) [reply assets/chat/js/formatters.js]: Pardon my unfamiliarity with GitHub, "requested changes" is akin to waiting for other's feedback and not for me to remove the code so you can merge it the code you want, ye?

Comment (jbpratt78) [reply assets/chat/js/formatters.js]: You are fine. I am waiting for others that have been requested to take a look and decide. After they review the PR then we can merge it in. If they do decide this feature is unwanted coupled with the modifier then we will ask you to remove it then we will merge. :+1:

**Comment by WhiteNervosa** [comment]
My apologies, I just now understood what JBPratt meant, I'll split all of these into separate branches so its easier for you guys later today. I'm learning Git, my apologies.

**Comment by xDashh** [comment]
I personally generally dislike taking the main meme of an existing emote and generifying it (as e.g. with `dank`), but I can also see some good meme potential out of this, so let's try I guess.

@jbpratt78 your changes were addressed? Also, is the timing the same as currently on `LAG` emote?

**Comment by jbpratt78** [comment]
@xDashh yep, looks good. I agree on the concern but am down to try it out.

**Comment by xDashh** [comment]
@jbpratt78 you should be able to approve and merge (if it makes sense to merge this before others, I'm not sure)

**Comment by jbpratt78** [comment]
Merged, thank you!

**Pull request merged**